src/bp/core/routers/admin/index.ts

```diff
@@ -10,7 +10,7 @@ import { RequestHandler, Router } from 'express'
 import _ from 'lodash'
 
 import { CustomRouter } from '..'
-import { checkTokenHeader, loadUser } from '../util'
+import { checkTokenHeader, loadUser, needSuperAdminAccess } from '../util'
```

Comment:
we already use assertIsRootAdmin and assertUserExist to check for conditions, we could use assertSuperAdmin instead (and also rename assertIsRootAdmin to assertAdmin at the same time)